driver-core/src/main/com/mongodb/internal/connection/ReadConcernHelper.java

```diff
@@ -14,42 +14,31 @@
  * limitations under the License.
  */
 
-package com.mongodb.operation;
+package com.mongodb.internal.connection;
 
-import com.mongodb.ReadConcern;
 import com.mongodb.ReadConcernLevel;
 import com.mongodb.session.SessionContext;
 import org.bson.BsonDocument;
 import org.bson.BsonString;
 
 import static com.mongodb.assertions.Assertions.notNull;
 
-final class ReadConcernHelper {
+public final class ReadConcernHelper {
 
-    static void appendReadConcernToCommand(final ReadConcern readConcern, final SessionContext sessionContext,
-                                           final BsonDocument commandDocument) {
-        notNull("readConcern", readConcern);
+    public static BsonDocument getReadConcernDocument(final SessionContext sessionContext) {
```

Comment:
Refactored this so that it's also usable in CommandMessage, now that readConcern has to be included in the first message of each transaction